### PR TITLE
feat: Explain why diego-cells should be placed on dedicated nodes.

### PR DESCRIPTION
**Pull request by andreas-kupries**
This implements
https://github.com/cloudfoundry-incubator/kubecf/issues/1287

### CODE REVIEW

**Comment by andreas-kupries** [comment]
CAP ticket https://jira.suse.com/browse/CAP-1567

File: content/en/docs/Deployment/kubernetes-deploy.md [hunk 1]
```diff
@@ -145,6 +145,24 @@ Kubernetes namespace.
...
 `--set features.eirini.enabled=true` when deploying kubecf from its chart.
 
+#### Diego Cell Affinities & Tainted Nodes
+
+Note that the `diego-cell` pods used by the Diego standard scheduler
+are
+
+  - privileged,
+  - use large local emptyDir volumes (i.e. require node disk storage),
+  - and set kernel parameters on the node.
```
Comment (mook-as) [inline content/en/docs/Deployment/kubernetes-deploy.md:155]: This is (partially) why they need to be privileged. Noting it here is probably too much detail, though.

File: content/en/docs/Deployment/kubernetes-deploy.md [hunk 2]
```diff
@@ -145,6 +145,24 @@ Kubernetes namespace.
...
+
+These things all mean that these pods should not live next to other
+Kubernetes workloads. They should all be placed on their own
+__dedicated nodes__ instead.
```
Comment (mook-as) [inline content/en/docs/Deployment/kubernetes-deploy.md:159]:
```suggestion
__dedicated nodes__ instead where possible.
```

**Review by mook-as — APPROVED** [review]

File: content/en/docs/Deployment/kubernetes-deploy.md (hunk 1 above)
Comment (andreas-kupries) [reply content/en/docs/Deployment/kubernetes-deploy.md:155]: Maybe rephrase, move the info about privileged to the line about kernel params?

Comment (mook-as) [reply content/en/docs/Deployment/kubernetes-deploy.md:155]: Nah, this is probably fine.

**Review by mook-as — APPROVED** [review]
